src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -82,7 +82,7 @@
                              RelativeSource="{RelativeSource  Mode=FindAncestor, AncestorType=UserControl}" />
                     <Binding>
                         <Binding.Source>
-                            <sys:Double>-40.0</sys:Double>
+                            <sys:Double>0</sys:Double>
```

Comment:
whats this?

Reply:
Tested it and this conversion makes 0 sense with a 0. :D I reworked the whole bit and removed the redundant code.